docs/how-to-guides/creating-grc20.md

```diff
@@ -3,137 +3,142 @@ id: creating-grc20
 ---
 
 # How to create a GRC20 Token
-
 ## Overview
 
-This guide shows you how to write a simple _GRC20_ Smart Contract, or rather a [Realm](../concepts/realms.md), in [Gno (Gno)](../concepts/gno-language.md). For actually deploying the Realm, please see the [deployment](deploy.md) guide.
+This guide shows you how to write a simple **GRC20** Smart Contract, or rather
+a [Realm](../concepts/realms.md), in [Gno](../concepts/gno-language.md). For actually deploying the Realm, please see the
```

Comment:
also should realm be capitalized? If not, just a few places that we can change to lowercase.

Reply:
This is a bit hard to decide on - sometimes it makes sense to have it capitalized, but sometimes it just doesn't feel right. Not sure what to do on this.

Reply:
I'll make a separate PR for this. I think it would be alright to keep realm/package lowercase most of the time, except when using it in titles, and maybe when referring to them as concepts.